app/models/case_log.rb

```diff
@@ -124,6 +124,10 @@ def net_income_soft_validation_triggered?
     net_income_in_soft_min_range? || net_income_in_soft_max_range?
   end
 
+  def rent_in_soft_validation_range?
```

Comment:
Looks like these methods don't actually get called at all. Were they left in by mistake?